tests/test_defects.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env python
+
+#
+# LSST Data Management System
+#
+# Copyright 2008-2017  AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+"""Test cases for cp_pipe."""
+
+from __future__ import absolute_import, division, print_function
+import unittest
+import numpy as np
+import copy
+
+import lsst.utils
+import lsst.utils.tests
+
+import lsst.cp.pipe as cpPipe
+import lsst.ip.isr.isrMock as isrMock
+from lsst.afw.geom import Box2I, Point2I, Extent2I
+from lsst.cp.pipe.utils import countMaskedPixels
```

Comment:
Please order these so the cp pipe stuff goes together. I've started to settle on ordering import blocks alphabetically, but I'm not wedded to the idea. But at least keep similar bits grouped.